guacamole/src/main/frontend/src/app/settings/templates/settingsConnections.html

```diff
@@ -10,7 +10,7 @@
         <div class="action-buttons">
 
             <a class="import-connections button"
-                ng-show="canCreateConnections()"
+                ng-show="canAdminister()"
```

Comment:
Changed this to only show for administrators, due to the wide variety of permissions actually required to do the import.